README.md

```diff
@@ -34,6 +34,9 @@ z wprowadzeniem odpowiednich zmian również do innych wersji.
 * W szablonie wiadomości e-mail z powiadomieniem o wystawieniu nowej faktury dodaj `%billtech_btn` i/lub `%billtech_balance_btn` w miejscu,
   w którym powinny pojawić się przyciski do opłacenia odpowiednio indywidualnej faktury i/lub salda. 
 
+## Odinstalowanie wtyczki
+Proces odinstalowania wtyczki został opisany w oddzielnym pliku [UNINSTALL.md](../blob/master/UNINSTALL.md).
```

Comment:
Można jeszcze dodać, że to jest instrukcja kompletnego odinstalowywania wtyczki. Czasami chcemy ją tylko wyłączyć na jakiś czas. Wtedy zmienne konfiguracyjne, tabele bazy danych i inne mogą zostać. Jedyny problem z tym to zaburzony porządek.

Reply:
@pablos-billtech poprawione